### ISSUE TITLE
Limits: 100 images per request, 200 points per image. (We will support 1000 points per image in the near future.)

### ISSUE THREAD

**Issue by Jordan-Pierce** [coralnet/coralnet#483]
Hi @StephenChan 

Just checking if this^ has been implemented, or if the limit is still 200 points per image? Follow up:  if say 201 points are provided in the request to CoralNet, does your side just ignore everything after the first 200 points and the last 201th point is dropped, or added to another job? How are those cases handled?

Thanks!

Jordan

**Comment by StephenChan** [comment]
Hi Jordan,

We did not yet change the limit from 200 to 1000. Good catch, I'd forgotten about that.

I just refreshed my memory on why the limit was lowered (issue #277, October 2020: compute nodes ran out of memory, but that was anticipated to not be a problem with the pyspacer version that was soon to be deployed then). So I'd definitely do some testing first to ensure that the current infrastructure is indeed fine with 1000 points.
